### DISCUSSION TITLE
Readme Update for Tags / Releases

### DISCUSSION THREAD

**Discussion by KleinerHacker** [openupm/openupm#2689] [Ideas]
I see that changes in readme.md is not observe in case I create a new tag. But I requires this cause my frameworks grown up with new features that are not shown in Unity itself. Please insert a readme.md update, too.

**Comment by favoyang** [comment]
Which package@version?

**Reply by favoyang** [reply]
Okay, the behavior of the view documentation button has been changed. It is used to locate the README.md at the root path. But it changed to follow either `Documentations~/ANY_NAME.md` or the documentationUrl in package.json.

See more in https://docs.unity3d.com/Manual/cus-document.html#doc_format and https://docs.unity3d.com/Manual/upm-manifestPkg.html#documentationUrl
